docs/_docs/04_reference/cli-options.md

```diff
@@ -1149,7 +1151,15 @@ When passed in, running `hclfmt` will print diff between original and modified f
 
 - [hclfmt](#hclfmt)
 
-When passed in, run `hclfmt` only on specified hcl file.
```

Comment:
You deleted the docs for `terragrunt-hclfmt-file`.

Reply:
Given that this PR has been going on for a very long time, I'm just going to merge this in, then fix the docs issue in a follow-up PR.

Thanks for your contribution @alikhil !